test/commands/markdown/parse_test.rb

```diff
@@ -6,8 +6,7 @@ class Markdown::ParseTest < ActiveSupport::TestCase
   end
 
   test "converts markdown to html" do
-    expected = '<h1>OHAI</h1>
-<p>So I was split between two ways of doing this.</p>
+    expected = '<p>So I was split between two ways of doing this.</p>
```

Comment:
I had to remove the h1 as that is now stripped by default. I felt that removing it here didn't invalidate the test.